### ISSUE TITLE
Automatic Reload Feature

### ISSUE THREAD

**Issue by Sh4d0wF1gh7** [redminenb/redminenb#48]
Hello,

perhabs a idea for an easy feature, can you add a functionality to automatic reload not only on startup the IDE, also during work? Often i work in parallel on web view and add or modify tickets.
An automatic reload every time x (setting in the query in minutes???) would be very nice.

Also an idea, if i checkin a revision and insert Subversion Information to the ticket, i have to reload it.
On this step, after modifieng the ticket from checkin a reload would also be great.

Greets

**Comment by matthiasblaesing** [comment]
PRs welcome

**Comment by RolandSM** [comment]
PRs welcome??? means???

**Comment by matthiasblaesing** [comment]
PR => Pull Request

I meant to say: If someone wants to implement this, he should do it. I'm willing to review and merge. I most probably won't do it.
